components/central-application-gateway/internal/proxy/reverseproxy.go

```diff
@@ -54,7 +55,10 @@ func newProxy(targetURL string, requestParameters *authorization.RequestParamete
 
 		log.Infof("Modified request url : '%s', schema : '%s', path : '%s'", req.URL.String(), req.URL.Scheme, req.URL.Path)
 	}
-	return &httputil.ReverseProxy{Director: director, Transport: transport}, nil
+	errorHandler := func(rw http.ResponseWriter, req *http.Request, err error) {
+		codeRewriter(rw, err)
+	}
```

Comment:
Aren't we missing logging now?

Reply:
I will add necessary logging 👍🏻

Reply:
And what about logging other errors?
Also we *could* take into account `ErrorLog` field of the `ReverseProxy`, 
but I don't feel like it's really necessary if we're not configuring that field.

Reply:
Other errors are handled by a default error handler in the reverse proxy package.